manifests/RProject/R/4.0.0.yaml

```
@@ -0,0 +1,12 @@
+Id: RProject.R
+Publisher: RProject
+Name: RProject R
```

I would add `AppMoniker` as the specification defines it by `# The common name someone may use to search for the package.`
```suggestion
Name: RProject R
AppMoniker: R
```